### PR TITLE
Mobile-view

### PR DESCRIPTION
**Pull request by Gimbalgal**
Design a mobile view of a traveling website. I am sending a pull request for evaluation and correction.

Thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        <img src="image/Rhine Falls.png" alt="">
+        <h2>Rhine Falls</h2>
+        <div class="location">
+            <img src="image/Location.svg" alt="">
+        <h3>Jawa Tengah, Indonesia</h3>
+        </div>
+        </div>
+    </div>
+
+    <div class="Sunter Lake ">
```
Comment (Devsalmasaeed) [inline index.html]: - Remove white space in your code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    <div class="section">
+        <div class="listtop"></div>
+        <h1>Best Guide, <br> Best Experience</h1>
+        <p>Your trip will be very memorable with the <br> best guide.</p>
+        <div class="detail">
+            <h2>more details</h2>
+            <img src="image/arrow-right b.svg" class="arrow-right"  alt="">
+        </div>
+    </div>
+
```
Comment (Devsalmasaeed) [inline index.html]: - Remove white space

File: index.html [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    <div class="trip">
+        <h1>Watch our entire <br> trip <br> documentation</h1>
+            <div class="video-container">
+                <video  class="video" src="Video/travel video.mp4"></video>
+            </div>
+    </div>
+</div>
+<!--end of video-->
+
+<div class="footer">
```
Comment (Devsalmasaeed) [inline index.html]: - Remove white space

**Review by Devsalmasaeed — APPROVED** [review]
Hi @Gimbalgal,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    Remote Collaboration
</div>

### Highlights
- ✅ **Professional Readme**
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Gilda+Display&family=Urbanist:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    
+</head>
+<body>
+    
+    
+        <div class="nav">
```
Comment (Xfarouq) [inline index.html]: Avoid leaving spaces in your code for best practices

**Review by Xfarouq — CHANGES_REQUESTED** [review]
Hi @username,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

- Please make the section below to be centrally aligned with the whole website to avoid it overflowing (i. e moving horizontally left to right)
- 
image


### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — APPROVED** [review]
> Hi @Gimbalgal,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it!!! Congratulations! 🎉
> 
> Remote Collaboration
> ### Highlights
> * ✅ **Professional Readme**
> * ✅ **Good use of Gitflow**
> * ✅ **UI looks like the Figma design**
> * ✅ **Good use of HTML semantics**
> * ✅ **Professional commit messages**
> * ✅ **Professional PR title**
> * ✅ **Professional PR description**
> 
> ### Optional Suggestions
> Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
> 
> Cheers and Happy coding! 👏👏👏
> 
> _Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification._

**Pull request merged**